ui/src/main/java/com/xmartlabs/base/ui/SingleFragmentWithToolbarActivity.java

```diff
@@ -1,18 +1,16 @@
-package com.xmartlabs.template.ui;
+package com.xmartlabs.base.ui;
 
 import android.os.Bundle;
 import android.support.annotation.NonNull;
 import android.support.annotation.Nullable;
 import android.support.v7.widget.Toolbar;
 
-import com.xmartlabs.template.R;
-
 import butterknife.BindView;
 import butterknife.ButterKnife;
 
 /** {@link SingleFragmentActivity} with {@link Toolbar} support. */
 public abstract class SingleFragmentWithToolbarActivity extends SingleFragmentActivity {
-  @BindView(R.id.toolbar)
+  @BindView(R2.id.toolbar)
```

Comment:
I think it is for security reasons from what I have researched about how it was done. Don't remember specifically why now.